tests/typechecking/base_model.py

```diff
@@ -28,3 +28,6 @@ class Knight(BaseModel):
 
 
 k = Knight()  # type: ignore[call-arg]  # pyright: ignore[reportCallIssue]
+
+MyModel.model_fields.keys()
+MyModel.model_computed_fields.keys()
```

Comment:
```suggestion
assert_type(Knight.model_fields, dict[str, FieldInfo])
assert_type(Knight.model_computed_fields, dict[str, ComputedFieldInfo])
assert_type(k.model_fields, dict[str, FieldInfo])
assert_type(k.model_computed_fields, dict[str, ComputedFieldInfo])
```

importing `assert_type` from `typing_extensions`